packages/d/directxshadercompiler/xmake.lua

```diff
@@ -7,16 +7,19 @@ package("directxshadercompiler")
     local date = {["1.5.2010"] = "2020_10-22",
                   ["1.6.2104"] = "2021_04-20",
                   ["1.6.2106"] = "2021_07_01",
-                  ["1.7.2212"] = "2022_12_16"}
+                  ["1.7.2212"] = "2022_12_16",
+                  ["1.8.2403.2"] = "2024_03_29"}
     if is_plat("windows") then 
         add_urls("https://github.com/microsoft/DirectXShaderCompiler/releases/download/v$(version).zip", {version = function (version) return version .. "/dxc_" .. date[tostring(version)] end})
         add_versions("1.5.2010", "b691f63778f470ebeb94874426779b2f60685fc8711adf1b1f9f01535d9b67f8")
         add_versions("1.6.2104", "ee5e96d58134957443ded04be132e2e19240c534d7602e3ab8fd5adc5156014a")
         add_versions("1.6.2106", "053b2d90c227cae84e7ce636bc4f7c25acd224c31c11a324885acbf5dd8b7aac")
         add_versions("1.7.2212", "ed77c7775fcf1e117bec8b5bb4de6735af101b733d3920dda083496dceef130f")
+        add_versions("1.8.2403.2", "500332ff6e6f2739910239937016d4e9cccdc0d2251a33a94e366bdf18c1d5fd")
```

Comment:
```
=> download https://github.com/microsoft/DirectXShaderCompiler/releases/download/v1.7.2212/dxc_2022_12_16.zip .. ok
checking for Microsoft Visual Studio (x64) version ... 2019
finding directxshadercompiler from xmake ..
checking for xmake::directxshadercompiler ... directxshadercompiler 1.7.2212
{ 
  linkdirs = { 
    "C:\Users\runneradmin\AppData\Local\.xmake\packages\d\directxshadercompiler\1.7.2212\b164fab3037445[979](https://github.com/xmake-io/xmake-repo/actions/runs/8948475633/job/24581714764#step:5:980)61d832463393d6a\lib", 
  },
  sysincludedirs = { 
    "C:\Users\runneradmin\AppData\Local\.xmake\packages\d\directxshadercompiler\1.7.2212\b164fab303744597961d832463393d6a\include", 
  },
  version = "1.7.2212",
  license = "LLVM",
  static = true,
  libfiles = {
```

It appears that the CI is still testing the old version.

Reply:
please use `1.8.2403` instead of `1.8.2403.2`

Reply:
done